phase1/vsphere/README.md

```diff
@@ -120,15 +120,15 @@ number of nodes (phase1.num_nodes) [4] (NEW) 4
 ```
 cluster name (phase1.cluster_name) [kubernetes] (NEW) kubernetes
 ```
-* phase1.ssh_user - This field is not used for vSphere Deployment. Leave it blank and hit enter.
 
+* phase1.ssh_user - This field is not used for vSphere Deployment. Leave it blank and hit enter.
 ```
-SSH user to login to OS for provisioning (phase1.ssh_user) [] (NEW) 
+SSH user to login to OS for provisioning (phase1.ssh_user) [] (NEW)
 ```
 
 * Select the provider, in this case it would be vsphere.
 ```
-cloud provider: gce or vsphere (phase1.cloud_provider) [vsphere] (NEW) vsphere
+cloud provider: gce, azure or vsphere (phase1.cloud_provider) [vsphere] (NEW) vsphere
```

Comment:
@jjasghar Currently, there is no support for azure cloudprovider in k8s-anywhere. Can you please change this?

Reply:
I copy pasted this directly from the install script, is that wrong?

Reply:
@jjasghar You are right. My comment was for ```make docker-dev```. Yet, this is for the container ```docker run -it -v /tmp:/tmp --rm --env="PS1=[container]:\w> " --net=host cnastorage/kubernetes-anywhere:latest /bin/bash``` which is mentioned in README